docs/ca/dev/spryker-monitoring-integration.md

```diff
@@ -0,0 +1,81 @@
+
+## Telemetry data in scope of Spryker Monitoring Integration
+
+The Spryker Monitoring Integration focuses on the following entities to provide comprehensive monitoring.
+
+### Traces and spans
+
+{% info_block infoBox %}
+The supported version of OTel Semantic Conventions is 1.30.0.
+{% endinfo_block %}
+
+In OpenTelemetry, a *trace* represents the journey of a single request or transaction as it moves through various components of a system, capturing the end-to-end flow. A *span* is a single operation or unit of work within a trace, containing information like the operation name, start and end times, and any relevant metadata. Together, traces and spans provide a detailed view of the interactions and performance of different parts of an application, helping to diagnose issues and optimize performance.
+
+### Health status metrics
+Spryker Monitoring Integration gives access to a set of service health metrics. These metrics provide a high-level view into the health status of enabled services such us database, message broker, or scheduler. The level of detail is as follows.
+
+{% info_block infoBox %}
+The terms labels, dimensions, and attribute are used interchangeably in relation to metrics depending on the telemetry platform used. For example, in Grafana, you have labels, and in Dynatrace, you have dimensions.
+{% endinfo_block %}
+
+
+#### Metrics without dimensions
+The following metrics return a gauge with a binary value: 1 for green and 0 for red. They are composites of multiple service-related signals that, after transformations and calculations, yield a single value indicating the service's health.
+
+```yaml
+hc_rds: Reports 0 or 1 for the overall health of the RDS Service.
+hc_jenkins: Reports 0 or 1 for the overall health of the Jenkins Service.
+hc_rabbitmq: Reports 0 or 1 for RabbitMQ health.
+```
+
+#### Metrics with dimensions
+These metrics have the following labels:
+```yaml
+hc_rabbitmq_message_count_sum: A count of RabbitMQ messages [dimension_queue, dimension_virtualhost].
+hc_jenkins_builds_success_build_count_total.count: A count of successful Jenkins jobs [jenkins_job].
+hc_jenkins_builds_failed_build_count_total.count: A count of failed Jenkins jobs [jenkins_job].
+hc_tasks_cpu_average: CPU utilization in % for cluster tasks [dimension_clustername, dimension_servicename].
+hc_tasks_memory_utilization: Memory utilization in % for cluster tasks [dimension_clustername, dimension_servicename].
+```
+
+#### Metric details
+The default metrics resolution is 60 seconds. All metrics can be split by telemetry-data-account.
+
+## How do I get it?
```

Comment:
get what?

Reply:
Spryker Monitoring Integration. Any suggestions how to rephrase it?